### DISCUSSION TITLE
请问代码高亮是用哪个库做的

### DISCUSSION THREAD

**Discussion by VanLiuZhi** [HEIGE-PCloud/DoIt#113] [Q&A] [answered]
这个代码高亮效果非常nice，我也不是主攻前端的，一时半会看不出来，麻烦知道的朋友告诉一下

**Comment by HEIGE-PCloud** [comment, marked as answer]
没有使用第三方库，相关的 CSS 在这两个文件中：

https://github.com/HEIGE-PCloud/DoIt/blob/main/assets/css/_partial/_single/_code.scss
https://github.com/HEIGE-PCloud/DoIt/blob/main/assets/css/_variables.scss

Tokenization 是由 Hugo 处理的：

https://gohugo.io/getting-started/configuration-markup/#highlight
https://gohugo.io/content-management/syntax-highlighting/
